forge-gui/res/cardsfolder/upcoming/nemata_primeval_warden.txt

```diff
@@ -0,0 +1,14 @@
+Name:Nemata, Primeval Warden
+ManaCost:2 B G
+Types:Legendary Creature Treefolk
+PT:3/4
+K:Reach
+R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Battlefield | Destination$ Graveyard | ValidLKI$ Card.Creature+nonToken+OppOwn | ReplaceWith$ Exile | Description$ If a nontoken creature an opponent controls would die, exile it instead. When you do, create a 1/1 green Saproling creature token.
+SVar:Exile:DB$ ChangeZone | Hidden$ True | Origin$ All | Destination$ Exile | Defined$ ReplacedCard | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | Execute$ TrigToken | TriggerDescription$ If a nontoken creature an opponent controls would die, exile it instead. When you do, create a 1/1 green Saproling creature token.
```

Comment:
Need conditions for this:
> If a creature an opponent controls would die, and both Nemata and another effect are trying to exile it
instead, that opponent chooses which effect to apply. If Nemata’s effect ends up not applying because of
this choice, the triggered ability that creates a Saproling token won’t trigger, even if the creature card ends
up in exile.

Reply:
I added a `ConditionDefined$ ReplacedCard | ConditionPresent$ Card` check to make sure the triggered ability only fires if Nemata's ability governed the exile replacement effect. Notably, I initially had borrowed the first part of Valentin, Dean of the Vein's script when writing Nemata's, and Valentin does not have a similar check. Should this be addressed in a later PR?

Reply:
Hmm I will check in a bit
I might have been wrong and it will be fine without

Reply:
Further testing is showing me that Nemata's, Valentin's, and Kalitas, Traitor of Ghet's exile effects do not conflict with each other either; each effect will make the appropriate token if chosen first, and the other effects will not override it. I'll tentatively resolve this topic with Nemata's safeguard addition intact. I'm happy to revisit this if we do discover a break case

Reply:
I think for now you can remove it again, there shouldn't be anything yet that can replace the exiling
Sorry for that

Reply:
No worries, I'll revert the change. Thank you for all your feedback so far